### ISSUE TITLE
Use new default HTTPS configuration on github with custom domains

### ISSUE THREAD

**Issue by emileber** [saglacio/saglac.io-original#157]
https://blog.github.com/2018-05-01-github-pages-custom-domains-https/

**Comment by jipiboily** [comment]
yup, I've seen it. Is there a reason to move to it? Outside of not depending on my Cloudflare account (which we depend on for DNS anyway)?

**Comment by emileber** [comment]
> not depending on my Cloudflare account

Mostly for this.

**Comment by jipiboily** [comment]
We will still depend on it, though. As this is where the DNS are hosted.

I don’t have time right now, though.

**Comment by emileber** [comment]
There's no rush for this 😉
